Autopilot/Autopilot.Windows/Autopilot.Windows.csproj

```diff
@@ -148,12 +142,21 @@
       <SubType>Designer</SubType>
     </Page>
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Data.Entity, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL" />
```

Comment:
Das kann raus, wegen des Kommentars in Zeile 108. Ich hab das grad mal versucht zu bauen und das gesamte .NET ist ohne Weiteres verfügbar. Einfach includen^^